modules/installation-gcp-config-yaml.adoc

```diff
@@ -118,6 +123,9 @@ For production {product-title} clusters on which you want to perform installatio
 ifdef::private[]
 <10> How to publish the user-facing endpoints of your cluster. Set `publish` to `Internal` to deploy a private cluster, which cannot be accessed from the internet. The default value is `External`.
 endif::private[]
+ifdef::user-infra-vpc[]
+To use an existing VPC in a cluster that uses infrastructure that you provision, you must set `publish` to `Internal`. The installation program will no longer be able to access the public DNS zone for the the base domain in the host project.
```

Comment:
I suggest to replace "an existing" with "a shared". Shared VPC stands for cross project VPC which is not quite same as existing VPC